loadtest/control/simulcontroller/controller.go

```diff
@@ -234,6 +234,18 @@ func (c *SimulController) Run() {
 			run:       c.getInsights,
 			frequency: 0.011,
 		},
+		{
+			run:       control.CreateAckPost,
+			frequency: 0.225,
+		},
+		{
+			run:       control.AckToPost,
+			frequency: 0.5,
+		},
+		// {
+		// 	run:       control.CreatePersistentNotificationPost,
+		// 	frequency: 5,
+		// },
```

Comment:
Are these numbers based on real data? You can follow https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/coverage-frequency.md for coming up with the actual numbers.
Also: why is the last action commented out?

Reply:
@koox00 Do we have real no. for Acks as well?

@agarciamontoro  The commented section is actually dependent on the bug I mentioned in the `Random` func. https://mattermost.atlassian.net/browse/MM-49975

Reply:
Sorry, I haven't had time to look into that bug yet. On the frequency for persistent notification, we can start with a very low number, then, and adjust when we have real data on it.

Reply:
Btw, I'd love for you to also use the `MinServerVersion` field for the new actions. I know it's not anywhere else, but I've been asking for it for new actions, and I'll take care of old ones to set it.